src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/TruncateCollectionOperation.java

```diff
@@ -0,0 +1,30 @@
+package io.stargate.sgv2.jsonapi.service.operation.model.impl;
+
+import com.datastax.oss.driver.api.core.cql.SimpleStatement;
+import io.smallrye.mutiny.Uni;
+import io.stargate.sgv2.jsonapi.api.model.command.CommandContext;
+import io.stargate.sgv2.jsonapi.api.model.command.CommandResult;
+import io.stargate.sgv2.jsonapi.service.cqldriver.executor.QueryExecutor;
+import io.stargate.sgv2.jsonapi.service.operation.model.Operation;
+import java.util.function.Supplier;
+
+/**
+ * Implementation of the truncate collection.
+ *
+ * @param context Command context, carries namespace and collectoin of the collection.
```

Comment:
nit: "collectoin" -> "Collection". But also... not sure what "collection of the collection" means? Maybe just "name of the collection"?

Reply:
:+1: this comment is unclear